contracts/tunnel-consumer/src/state.rs

```diff
@@ -0,0 +1,55 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{IbcChannel, IbcEndpoint, Uint64};
+use cw_controllers::Admin;
+use cw_storage_plus::Map;
+
+use cw_band::tunnel::packet::Price;
+
+pub const ADMIN: Admin = Admin::new("admin");
+pub const TUNNEL_CONFIG: Map<&str, TunnelConfig> = Map::new("tunnel_config");
+pub const CHANNEL_INFO: Map<&str, ChannelInfo> = Map::new("channel_info");
```

Comment:
what are channelInfo state and channelInfo used for? I can't find where to use it.